umap/static/umap/js/umap.core.js

```diff
@@ -433,6 +433,64 @@ L.U.Keys = {
 }
 
 L.U.Help = L.Class.extend({
+
+  SHORTCUTS: {
+    DRAW_MARKER: {
+      shortcut:'Modifier+M',
+      label: L._('Draw a marker')
+    },
+    DRAW_LINE: {
+      shortcut:'Modifier+L',
+      label: L._('Draw a polyline')
+    },
+    DRAW_POLYGON: {
+      shortcut:'Modifier+P',
+      label: L._('Draw a polygon')
+    },
+    TOGGLE_EDIT: {
+      shortcut:'Modifier+E',
+      label: L._('Toggle edit mode')
+    },
+    STOP_EDIT: {
+      shortcut:'Modifier+E',
+      label: L._('Stop editing')
+    },
+    SAVE_MAP: {
+      shortcut:'Modifier+S',
+      label: L._('Save map')
+    },
+    IMPORT_PANEL: {
+      shortcut:'Modifier+I',
+      label: L._('Import data')
+    },
+    SEARCH: {
+      shortcut:'Modifier+F',
+      label: L._('Search location')
+    },
+    CANCEL: {
+      shortcut:'Modifier+Z',
+      label: L._('Cancel edits')
+    },
+    PREVIEW: {
+      shortcut:'Modifier+E',
+      label: L._('Back to preview')
+    },
+    SAVE: {
+      shortcut:'Modifier+S',
+      label: L._('Save current edits')
+    },
+  },
+
+  displayLabel: function (action, withKbdTag=true) {
+    let {shortcut, label} = this.SHORTCUTS[action]
+    if (withKbdTag) {
+      shortcut = shortcut.split('+').map((el) => `<kbd>${el}</kbd>`).join('+')
+    }
+    const modifier = this.isMacOS ? 'Cmd' : 'Ctrl'
+    label += ` ${shortcut.replace('Modifier', modifier)}`
```

Comment:
```suggestion
    label += ` (${shortcut.replace('Modifier', modifier)})`
```

I think keeping the keyboard shortcut between parenthesis is more readable.

Reply:
Once the `kbd` tag is styled, I find the parenthesis noisy:

![image](https://github.com/umap-project/umap/assets/146023/8c26e881-3f68-43ed-806a-affd7b30e4eb)


vs 

![image](https://github.com/umap-project/umap/assets/146023/d36b6ec4-0655-4f2d-b916-b866230bb1e9)

What do you think ?